pegasus/router.rb

```diff
@@ -226,7 +226,8 @@ def self.set_max_age(type, default)
 
   # Documents
   get_head_or_post '*' do |uri|
-    pass unless path = resolve_document(uri)
+    path = resolve_document("#{uri}.#{request.locale}") || resolve_document(uri)
```

Comment:
Whenever I see two possible code paths (separated by the `||` here) I think a comment could be helpful to explain the difference between the primary and the fallback path, and what would cause one to hit versus the other.